librz/core/cmd/cmd_debug.c

```diff
@@ -5182,3 +5088,14 @@ RZ_IPI int rz_cmd_debug_continue_until(void *data, const char *input) {
 	rz_core_dbg_follow_seek_register(core);
 	return RZ_CMD_STATUS_OK;
 }
+
+#define CMD_REGS_PREFIX   debug
+#define CMD_REGS_REG_PATH dbg->reg
+static bool cmd_regs_sync(RzCore *core, RzRegisterType type, bool write) {
+	return rz_debug_reg_sync(core->dbg, type, write);
+}
+#define CMD_REGS_SYNC cmd_regs_sync
+#include "cmd_regs_meta.inc"
+#undef CMD_REGS_PREFIX
+#undef CMD_REGS_REG_PATH
+#undef CMD_REGS_SYNC
```

Comment:
you haven't added `dr` commands in the yaml files yet, did you? You are going to add them, if I get this right?

Reply:
Exactly, they are still missing in the yaml. When they are there, their handlers will point to the macro-generated functions here.